engine/security/src/main/scala/pl/touk/nussknacker/ui/security/LoggedUser.scala

```diff
@@ -1,7 +1,7 @@
-package pl.touk.nussknacker.ui.security.api
```

Comment:
What do you thing about having all this api stuff in api package and other implementation have in security/module e.g. security/oauth?